src/RegistryCoordinator.sol

```diff
@@ -695,6 +695,13 @@ contract RegistryCoordinator is
         }
     }
 
+    /**
+     * @notice Returns true iff all of the given quorum exists
+     */
+    function _quorumExists(uint8 quorumNumber) internal view returns (bool) {
```

Comment:
If we use this, might be cleaner to just scrap the `modifier _quorumExists` and use the function version.
The modifier is only being called in `setOperatorSetParams`